carrie_gz/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Not really important, but we should strive to use the latest versions of CMake

Reply:
Oh, yes, I meant to comment on this. This needs to be updated to whatever is default on Ubuntu 24.04